target/scripts/startup/setup-stack.sh

```diff
@@ -95,19 +102,44 @@ function _setup_rspamd
 
   if [[ ${ENABLE_AMAVIS} -eq 1 ]] || [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]]
   then
-    _shutdown 'You cannot run Amavis/SpamAssassin and Rspamd at the same time'
+    _log 'warn' 'Running Amavis/SpamAssassin & rspamd at the same time is discouraged'
   fi
 
   if [[ ${ENABLE_CLAMAV} -eq 1 ]]
   then
     _log 'debug' 'Rspamd will use ClamAV'
-    mv /etc/rspamd/local.d/disabled/antivirus.conf /etc/rspamd/local.d/antivirus.conf
+    usermod -a -G clamav _rspamd
+    sedfile -i -E 's|^(enabled).*|\1 = true;|g' /etc/rspamd/local.d/antivirus.conf
   else
     _log 'debug' 'Rspamd will not use ClamAV (which has not been enabled)'
   fi
 
-  _log 'warn' 'Only running with default configuration'
-  _log 'warn' 'You will need to adjust the Postfix configuration yourself to use Rspamd as of now'
+  declare -a DISABLE_MODULES
+  DISABLE_MODULES=(
+    clickhouse
+    dkim_signing
+    elastic
+    greylist
+    rbl
+    reputation
+    spamassassin
+    url_redirector
+    metric_exporter
+  )
+
+  for MODULE in "${DISABLE_MODULES[@]}"
+  do
+    cat >"/etc/rspamd/local.d/${MODULE}.conf" << EOM
+#documentation: https://rspamd.com/doc/modules/${MODULE}.html
+
+enabled = false;
+
+EOM
```

Comment:
Is `EOM` different to the common `EOF` used for heredoc syntax elsewhere in the project? Otherwise perhaps it should be `EOF` here too?

Reply:
For the heredoc, I doesn't make any difference. But I will streamline it to `EOF` :D

Reply:
Done in 2b40f7848c53b0772262dcda04582d6a0f2162f7